apps/api/.env.example

```diff
@@ -1,7 +1,7 @@
 API_HOST=0.0.0.0
 API_PORT=3000
 
-DATABASE_HOST=mysql-dev
+DATABASE_HOST=mysql-resource
```

Comment:
With a change like this, what's the best way to notify everyone to update their local `.env` file?

Reply:
I'm really not sure, there's really no way to automate this process so I think the simplest way to just to @everyone and refrain from making these changes too often.